internal/cmd/billing/command_cost_list.go

```diff
@@ -0,0 +1,107 @@
+package billing
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+type command struct {
+	*pcmd.AuthenticatedCLICommand
+}
+
+type costOut struct {
```

Comment:
There are a lot of fields here... it seems likely that this will cause line wrapping on small terminals which could be a minor annoyance. Could any of these fields be removed or are they all important to the end user?

Reply:
@brianstrauch  is there a way to separate critical fields from verbose fields? I believe all these fields would be important to customers.

Reply:
We don't have anything like that. Typically the most important fields are ordered first. I'll suggest some renamings to make the field names shorter, which should save some space.